.travis.yml

```diff
@@ -21,12 +21,22 @@ before_install:
   - sudo rm -rf /usr/local/.git/refs /usr/local/.git/packed-refs
   - sudo rsync -az "$TRAVIS_BUILD_DIR/" /usr/local/
   - export TRAVIS_BUILD_DIR="/usr/local"
-  - sudo chown -R $USER /usr/local
-  - env | grep TRAVIS_
-  - if [ -f ".git/shallow" ]; then travis_retry git fetch --unshallow; fi
+  - if [ -f ".git/shallow" ]; then
+      travis_retry git fetch --unshallow;
+    fi
   - git reset --hard $TRAVIS_COMMIT
   - git clean -qxdff
 
+install:
+  - if [ "$HOMEBREW_OLD_RUBY_TEST" = "1.8.7" ]; then
+      brew install --build-bottle homebrew/versions/ruby187;
+      export PATH="/usr/local/opt/ruby187/bin:$PATH";
+      tar xf https://rubygems.org/rubygems/rubygems-2.5.0.tgz;
+      cd rubygems-2.5.0;
+      ruby setup.rb;
+      cd /usr/local
```

Comment:
There's a `;` missing here, but I guess you know already and instead focus on bringing `homebrew/versions/ruby187` in a more useful shape.